CHANGELOG.md

```diff
@@ -11,7 +11,12 @@ All changes that impact users of this module are documented in this file, in the
 - **Breaking:** In exposed CLI `--terms-types` option is now called `--types`. Short option is stil `-t`.
 - **Breaking:** In exposed API `filter` function is now called `extract`.
 - **Breaking:** In exposed API `PageDeclaration` class is now called `SourceDocument`.
-
+- **Breaking:** Change commit messages in Git data layer but keep backwards compatibility:
```

Comment:
If backwards compatibility is maintained, then this is not a breaking change 🙂 
In this case, I believe we should be explicit that the “backwards compatibility” is about reading from existing data repositories.

Reply:
If think it's a breaking change in the way that newly created data will have different commit messages and if the user created a script that parses commit messages it won't work anymore (it's the same for mongo document attributes). The backwards compatibility is only ensure inside the engine but data structure will change.

Reply:
Yes, I agree with you. I am sorry, I was not clear. I meant that the current wording sounds contradictory. This detail that you add is important to add to the changelog.